ADA/ada.qmd

```diff
@@ -19,57 +17,292 @@ It brings together:
+\*\* auto-formatting is available for SQL and Python in Databricks, but not R  
+\*\*\* external users must have software that would allow them to open notebooks
+
+You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+---
+
+
+#### Accessing code from a Git repository
+
+---
+
+Databricks has inbuilt Git functionality, and you can save your notebooks and scripts inside Git repositories as with any other file formats. There is guidance on using Databricks with Azure DevOps on our [Databricks and version control page](ADA/git_databricks.html). 
 
 ---
 
-## Databricks code templates
+## What this means for existing code
 
-See examples and code snippets in R, SQL and Python in these [Databricks code template notebooks](https://adb-6882499576863257.17.azuredatabricks.net/browse/folders/3070168888967919?o=6882499576863257#)
+### SQL scripts
+
+---
+
+#### Scripts currently used in SSMS
+
+---
+
+::: {.callout-important}
+Once your team's data has been migrated, you will **not** immediately lose access to your existing data sources, so you will be able to dual run your code using data from SSMS and data from the Databricks unity catalog  to ensure that everything is working as it should be. Once you have confirmed that you are happy with the data in Databricks, your access to the legacy platform will be switched off. 
+:::
+
+If you currently use data and scripts stored in SSMS for your analysis, there will be small changes you need to make that vary depending on your current process.
+
+Your SQL code will need to be migrated over to Databricks to allow it to read from the Databricks SQL Warehouse and will require some minor changes to syntax as SSMS uses T-SQL and Databricks uses Spark SQL. The ADA team are currently developing a tool to allow you to easily convert T-SQL queries to Spark SQL syntax, and Microsoft Copilot can also be useful for helping you with code translation. All DfE laptops should provide you with access to Copilot as standard.  
+
+You can write new SQL code in the Databricks SQL Editor, which is very similar to SSMS. We recommend using the SQL Editor for short, ad-hoc SQL queries, such as those required to answer PQs. For longer or more complex SQL analysis we recommend using standalone scripts run from an RStudio project, or Databricks notebooks. 
+
+---
 
-# What ADA means for statistics publications
+#### SQL scripts in repositories or embedded in R scripts
+
+---
+
+These scripts do not need to be moved, but the syntax of the scripts or embedded code will need to be rewritten from T-SQL to Spark SQL.
+
+You will also need to change the connection source to allow the code to run. Rather than pointing to data held in current SQL databases, you will need to point towards the Databricks Delta Lake instead.
```

Comment:
Should we link here to the section on how you do that?

Reply:
We should! Done!